bundles/org.openhab.binding.openuv/src/main/java/org/openhab/binding/openuv/internal/handler/OpenUVBridgeHandler.java

```diff
@@ -122,13 +124,20 @@ private void initiateConnexion() {
                 String error = uvResponse.getError();
                 if (error == null) {
                     updateStatus(ThingStatus.ONLINE);
+                    updateProperty(KEY_VERIFIED, Boolean.TRUE.toString());
                     return uvResponse.getResult();
                 }
                 throw new OpenUVException(error);
             }
         } catch (JsonSyntaxException e) {
-            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.COMMUNICATION_ERROR,
-                    String.format("Invalid json received when calling `%s` : %s", url, jsonData));
+            String message = "";
+            if (jsonData.contains("MongoError")) {
+                message = String.format("Service not responding, will reconnect in %d minutes", RECONNECT_DELAY_MIN);
+                scheduleReconnect(RECONNECT_DELAY_MIN);
+            } else {
+                message = String.format("Invalid json received when calling `%s` : %s", url, jsonData);
```

Comment:
JSON in capital letters and removal of space before colon:
```suggestion
                message = String.format("Invalid JSON received when calling `%s`: %s", url, jsonData);
```
This should be added as i18n text - see also line 147.